src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -454,7 +459,7 @@ trait BeamHelper extends LazyLogging {
     }), scala.concurrent.duration.Duration.Inf)
   }
 
-  def runBeamWithConfig(config: TypesafeConfig): (MatsimConfig, String) = {
+  def runBeamWithConfig(config: TypesafeConfig): (MatsimConfig, String, BeamServices) = {
```

Comment:
I probably missed it, but this was added and only seems to be ignored? What was the purpose of adding it?

Reply:
I added BeamServices to the return value in order to get them in ExperimentRunner#runExperiment. These services are needed for getting the output directory for evaluation.
https://github.com/LBNL-UCB-STI/beam/blob/051113d0cb20df7ceb1ce6c3ef2bfe30c64e2fb9/src/main/scala/beam/calibration/ExperimentRunner.scala#L53-L63